apps/core/src/migration/migration.e2e.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { ValidationPipe } from '@nestjs/common';
+import {
+  FastifyAdapter,
+  NestFastifyApplication,
+} from '@nestjs/platform-fastify';
+import { Test } from '@nestjs/testing';
+import { AppModule } from '../app.module';
+import { configureTestProcessEnv } from '../testing/aws-configuration';
+import { getAccessToken } from '../testing/jwt-generator';
+
+describe('DashboardsModule', () => {
+  let bearerToken = '';
+  let app: NestFastifyApplication;
+
+  beforeAll(async () => {
+    configureTestProcessEnv(process.env);
+
+    const moduleRef = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleRef.createNestApplication<NestFastifyApplication>(
+      new FastifyAdapter(),
+    );
+
+    app.useGlobalPipes(new ValidationPipe({ transform: true }));
+
+    await app.init();
+
+    const instance = app.getHttpAdapter().getInstance() as unknown as {
+      ready(): Promise<void>;
+    };
+
+    await instance.ready();
+
+    bearerToken = await getAccessToken();
+  });
+
+  afterAll(async () => {
+    await app.close();
+  });
+
+  describe('POST /api/migration HTTP/1.1', () => {
+    test('calls migration', async () => {
+      const response = await app.inject({
+        headers: {
+          Authorization: `Bearer ${bearerToken}`,
+        },
+        method: 'POST',
+        url: '/api/migration',
+      });
+
+      expect(response.statusCode).toBe(201);
```

Comment:
NestJS defaults to 201 on POST. What should the status code be?

Reply:
Since we're not going to be able to use websockets right now due to our app runner dependency, I'll need to update this API to handle async operations. So with that update I think it should return a 202 on the initial POST which signals `202 Accepted`. Then, a GET endpoint can be used to get the status of the migration, similar to what's described [here](https://restfulapi.net/http-status-202-accepted/)